tardis/montecarlo/montecarlo_numba/base.py

```diff
@@ -110,6 +112,9 @@ def montecarlo_radial1d(model, plasma, runner):
             np.array(virt_packet_last_line_interaction_out_id)
         ).ravel()
 
+    # Condition for Checking if R Packet Tracking is enabled
+    runner.r_packet_tracking = r_packet_track
```

Comment:
If this is a condition, I think the original variable `runner.r_packet_tracking` should be a boolean.

Reply:
I forgot about this. I was going to put a condition here such that the value will only be assigned if the tracking is turned on. The actual boolean storing the state of the tracking is `montecarlo_configuration.RPACKET_TRACKING`